sensor_msgs/RegionOfInterest region_of_interest
---
